packages/enzyme-utilities/README.md

```diff
@@ -56,3 +60,29 @@ describe('<MyComponent />', () => {
   });
 });
 ```
+
+### `function setupMount(): { mount, addMountedWrapper, unmountAllWrappers }
```

Comment:
Also, I feel like it's weird to have this here with no explanation of why we feel it's necessary. I know there's a bit of an explanation below about the manual usages, but if I was looking at using this package I don't think I'd be able to tell if / why I needed this.